test/packet/main.tf

```diff
@@ -46,6 +46,11 @@ resource "packet_device" "test" {
             destination="/provision"
     }
 
+    provisioner "file" {
+            source="../../contrib/scripts/add_vagrant_box.sh"
+            destination="/provision"
+    }
+
```

Comment:
@nebril Is this script still used? If not, should we remove it?

Reply:
@qmonnet Not used anymore in testing infra, but André and Chris use it. So we can't remove.

Reply:
Understood, thanks for the follow-up. I'll come back to this PR soon.